crates/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -385,9 +385,16 @@ fn debug_menu_options_ui(
     ).on_hover_text("Show a debug overlay that renders the picking layer information using the `debug_overlay.wgsl` shader.");
 
     re_ui.checkbox(ui,
-        &mut app_options.show_blueprint_in_timeline,
-        "Show Blueprint in the Time Panel",
-    ).on_hover_text("Show the Blueprint data in the Time Panel tree view. This is useful for debugging the internal blueprint state.");
+        &mut app_options.show_blueprint_timeline,
+        "Show a timeline of the blueprint data.",
```

Comment:
In the PR description this is called "blueprint inspection mode" which makes it sound more like a global mode, while "Show a timeline of the blueprint data." sounds like a change in one of the panels.

I think one problem is that while `show_blueprint_timeline` _mostly_ changes the streams panel, it _also_ does other things, like change how `current_query_for_entity_path` works.

Reply:
I think "blueprint inspection mode" is probably the better name and goes along with the explanation that while in this mode, the blueprint can't be edited.